code/modules/bitrunning/objects/landmarks.dm

```diff
@@ -68,3 +68,35 @@
 /obj/effect/landmark/bitrunning/safehouse_spawn
 	name = "Bitrunning safehouse spawn"
 	icon_state = "safehouse"
+
+///Swaps the locations of an encrypted crate in the area with another randomly selected crate.
+///Randomizes names, so you have to inspect crates manually.
+/obj/effect/landmark/bitrunning/crate_replacer
+	name = "Bitrunning Goal Crate Randomizer"
+	icon_state = "crate"
+
+/obj/effect/landmark/bitrunning/crate_replacer/Initialize(mapload)
+	. = ..()
+	var/list/crate_list = list()
+	var/obj/structure/closet/crate/secure/bitrunning/encrypted/encrypted_crate
+	for(var/obj/structure/closet/crate/crate_to_check in get_area(src))
```

Comment:
Iterating over every atom in an area is secretly an `in world` loop, which is not ideal. You should do something like:
```dm
var/area/my_area = get_area(src)
for(var/turf/area_turf as anything in my_area.get_contained_turfs())
    for(var/obj/structure/closet/crate/crate_to_check in area_turf)
        ...
```